lib_dsp/api/dsp_testing.h

```diff
@@ -17,19 +23,21 @@
  *
  * \returns                     Pseudorandom value of the type given in the function name.
  */
-int16_t  dsp_pseudo_rand_int16(unsigned *r);
-uint16_t dsp_pseudo_rand_uint16(unsigned *r);
-int32_t  dsp_pseudo_rand_int32(unsigned *r);
-uint32_t dsp_pseudo_rand_uint32(unsigned *r);
-int64_t  dsp_pseudo_rand_int64(unsigned *r);
-uint64_t dsp_pseudo_rand_uint64(unsigned *r);
-dsp_float_t dsp_pseudo_rand_float(unsigned *r);
-dsp_complex_int16_t dsp_pseudo_rand_complex_int16(unsigned *r);
-dsp_complex_int32_t dsp_pseudo_rand_complex_int32(unsigned *r);
-dsp_complex_float_t dsp_pseudo_rand_complex_float(unsigned *r);
-dsp_ch_pair_int16_t dsp_pseudo_rand_ch_pair_int16(unsigned *r);
-dsp_ch_pair_int32_t dsp_pseudo_rand_ch_pair_int32(unsigned *r);
-dsp_ch_pair_float_t dsp_pseudo_rand_ch_pair_float(unsigned *r);
+int8_t  dsp_pseudo_rand_int8(unsigned * UNSAFE r);
+uint8_t dsp_pseudo_rand_uint8(unsigned * UNSAFE r);
+int16_t  dsp_pseudo_rand_int16(unsigned * UNSAFE r);
+uint16_t dsp_pseudo_rand_uint16(unsigned * UNSAFE r);
+int32_t  dsp_pseudo_rand_int32(unsigned * UNSAFE r);
+uint32_t dsp_pseudo_rand_uint32(unsigned * UNSAFE r);
+int64_t  dsp_pseudo_rand_int64(unsigned * UNSAFE r);
+uint64_t dsp_pseudo_rand_uint64(unsigned * UNSAFE r);
+dsp_float_t dsp_pseudo_rand_float(unsigned * UNSAFE r);
+dsp_complex_int16_t dsp_pseudo_rand_complex_int16(unsigned * UNSAFE r);
+dsp_complex_int32_t dsp_pseudo_rand_complex_int32(unsigned * UNSAFE r);
+dsp_complex_float_t dsp_pseudo_rand_complex_float(unsigned * UNSAFE r);
+dsp_ch_pair_int16_t dsp_pseudo_rand_ch_pair_int16(unsigned * UNSAFE r);
+dsp_ch_pair_int32_t dsp_pseudo_rand_ch_pair_int32(unsigned * UNSAFE r);
+dsp_ch_pair_float_t dsp_pseudo_rand_ch_pair_float(unsigned * UNSAFE r);
```

Comment:
These are quite simple so probably don't need testing..? They're used only in test functions